magpie/ui/management/views.py

```diff
@@ -465,11 +465,17 @@ def get_user_or_group_resources_permissions_dict(self, user_or_group_name, servi
               or :term:`Group` accordingly to specified arguments.
         """
         if is_user:
-            query = "?inherited=true" if is_inherit_groups_permissions else ""
-            path = schemas.UserResourcesAPI.path.format(user_name=user_or_group_name) + query
+            # because page can only show a single permission (per name/resource) at a time, apply resolution
```

Comment:
What's the difference with before?
The single permission shown should be the effective, no?

Reply:
No. The displayed permission is either direct or inherited in the combobox, based on the checkbox "show intherited". 
The effective permission is obtained by the test button next to it that transforms into checkmark/cross. 

The problem was that given a resource X, a user member of both groups A and B, each with different permissions on X was not resolving the most important one to be displayed. 
Cases where `[(user, X, allow-match), (A, X, allowed-recurive), (B, X, deny-match)]` are applied would not be resolved and would simply show the last item in the HTML `<select> <options>` (that's how it resolves which one to show when many are selected for single-select comboboxes).

Reply:
Hum that's right, we don't have a "word" for the "most important one". But I meant the "local effective", without considering the scope. Anyway, we now have the good behavior. Thank you!

Reply:
Indeed there is no 'term' in the glossary, but I refer to that concept as `resolved` in docs.
It is present here: https://pavics-magpie.readthedocs.io/en/latest/permissions.html#types-of-permissions